doc/modules/model_evaluation.rst

```diff
@@ -91,6 +91,8 @@ Scoring                           Function
 'explained_variance'              :func:`metrics.explained_variance_score`
 'max_error'                       :func:`metrics.max_error`
 'neg_mean_absolute_error'         :func:`metrics.mean_absolute_error`
+'neg_mean_absolute_percentage_error'
+                                  :func:`metrics.mean_absolute_percentage_error`
```

Comment:
this will break rendered doc

Reply:
Please suggest the change. I have tried several ways but every time you have given this comment. Please suggest code snippet.